Examples/GetFinalPredictions/Program.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using IndicoV2;
+using Newtonsoft.Json.Linq;
+
+namespace Examples
+{
+    /// <summary>
+    /// Returns final output. Does not include pre_review and post_review predictions.

```

Comment:
This definitely returns predictions "post_review" and nothing else? vs. it returning only the raw results?

Reply:
it returns raw results (pre_review)